frappe/public/js/frappe/form/multi_select_dialog.js

```diff
@@ -272,19 +273,21 @@ frappe.ui.form.MultiSelectDialog = Class.extend({
 			query: this.get_query ? this.get_query().query : '',
 			as_dict: 1
 		}
+
 		frappe.call({
 			type: "GET",
-			method:'frappe.desk.search.search_widget',
+			method: method,
 			no_spinner: true,
-			args: args,
+			args: this.method ? {method_args: this.method_args} : args,
```

Comment:
method_args variable is not good, maybe override args's value if custom method has used.

Reply:
var d = new frappe.ui.form.MultiSelectDialog({
			doctype: "Material Request",
			target: this.frm,
			method: "dotted.path.to.method" ,
			**args**: {
			    field: value
			},
			setters: {
			    field: value
			},
		});

Reply:
args: this.args || args